src/Core/Core.csproj

```diff
@@ -34,9 +34,9 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.8.1908.202-beta" />
-    <PackageReference Include="Microsoft.Quantum.CsharpGeneration" Version="0.8.1908.202-beta" />
-    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.8.1908.202-beta" />
+    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.9.1908.2906" />
```

Comment:
It would be lovely if these version numbers were (possibly environment?) variables all set in one place.